src/devices/Max31865/README.md

```diff
@@ -0,0 +1,35 @@
+﻿# MAX31865 - Resistance Temperature Detector Amplifier
+The MAX31865 device is a SPI interface Resistance Temperature Detector to Digital Converter.
+
+## Sensor Image
+![](sensor.jpg)
```

Comment:
you can place a little description in the []. It's done for broken link and accessibility. Example:

![This text appear and explain the image and it's broken on purpose](./brokenlink.jpg)

Reply:
Similar comment for the few other places